app/styles/components/_aptables.scss

```diff
@@ -0,0 +1,106 @@
+//
+// Aptible Table Component
+//
+.aptable {
+  border: 1px solid #E0E0E0;
+  box-shadow: 0px 2px 4px 0px rgba(0,0,0,0.06);
+  border-radius: 3px;
+  border-collapse: separate;
+  width: 100%;
+
```

Comment:
It may be worth considering moving these styles to a parent container rather than directly on the table.  Many of our future tables include some metadata within the same container--unless we're going to do a `rowspan`, we'll need these styles on the parent container so we can pair the metadata with a table in the same container. 

e.g. https://www.dropbox.com/s/tv8ml0gq7kfc547/Screenshot%202016-06-13%2011.22.19.png?dl=0